script.js

```diff
@@ -0,0 +1,138 @@
+import { createCombineWinner } from './createCombineWinner.js'
+import { createComputerMove } from './computerMove.js'
+
+const main = document.querySelector('.main')
+
+export let gameBox = Array.from(new Array(9)).map(() => '')
+
+export let userSelection = null
+
+export const combineWinner = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+
+    [0,4,8],
+    [2,4,6]
+]
+
+const mainGame = document.createElement('div')
+mainGame.classList.add('main__game')
+mainGame.textContent = "Select Cross or Circle: "
+main.appendChild(mainGame)
+
+const game = document.createElement('div')
+game.classList.add('game')
+mainGame.appendChild(game)
+
+const gameSelectedItem = document.createElement('div')
+gameSelectedItem.classList.add('game__selected-item')
+game.appendChild(gameSelectedItem)
+
+const selectedButton = document.createElement('div')
+selectedButton.classList.add('selected')
+gameSelectedItem.appendChild(selectedButton)
+
+const buttonCross = document.createElement('div')
+buttonCross.classList.add('selected__button-cross')
+selectedButton.appendChild(buttonCross)
+
+const buttonCircle = document.createElement('div')
+buttonCircle.classList.add('selected__button-circle')
+buttonCircle.setAttribute("data-name","circle")
+selectedButton.appendChild(buttonCircle)
+
+const gameGrid = document.createElement('div')
+gameGrid.classList.add('game__grid')
+main.appendChild(gameGrid)
+
+const grid = document.createElement('div')
+grid.classList.add('grid')
+gameGrid.appendChild(grid)
```

Comment:
code above looks good, but I would wrap it in the function with proper name to make it more readable